### ISSUE TITLE
[Import] No such file 'binary'

### ISSUE THREAD

**Issue by Tigzy** [cuckoosandbox/cuckoo#1578]
Hello, 
I'm migrating from 2.0 RC1 to 2.0.2, and while migrating it crashes on the binary file

```
Traceback (most recent call last):
  File "/usr/local/bin/cuckoo", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 716, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 696, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 1060, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 889, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 534, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/cuckoo/main.py", line 570, in import_
    import_cuckoo(ctx.parent.user, mode, path)
  File "/usr/local/lib/python2.7/dist-packages/cuckoo/apps/import_.py", line 180, in import_cuckoo
    import_legacy_analyses(mode, dirpath)
  File "/usr/local/lib/python2.7/dist-packages/cuckoo/apps/import_.py", line 214, in import_legacy_analyses
    os.path.join(analyses, task_id), cwd(analysis=task_id)
  File "/usr/lib/python2.7/shutil.py", line 208, in copytree
    raise Error, errors
shutil.Error: [('cuckoo_old/storage/analyses/248/binary', u'/home/cuckoo/cuckoo/storage/analyses/248/binary', "[Errno 2] No such file or directory: 'cuckoo_old/storage/analyses/248/binary'")]
```

When I look into my 248 directory, the binary is a symlink that points to cuckoo/storage/binaries/some_hash, so I think it's just that import script doesn't resolve symlinks.

**Comment by Tigzy** [comment]
Ok, for some reason, it's the end binary (the symlink target) that doesn't exist. So the error looks legit however it would be nice to catch it.

**Comment by jbremer** [comment]
Yes, thanks. Probably the binary was deleted after analysis or so. I'll get it fixed :-)

**Comment by jbremer** [comment]
The above commit should fix the issue, just awaiting CI.

**Comment by Tigzy** [comment]
Thanks.

**Comment by jbremer** [comment]
Resolved as part of `2.0.3`, for this functionality you can upgrade using `pip install -U cuckoo`. Thanks!
